data/fields/maxspeed/variable.json

```
@@ -0,0 +1,17 @@
+{
+    "key": "maxspeed:variable",
+    "type": "combo",
+    "label": "Variable Speed Limit",
+    "strings": {
+        "options": {
+            "yes": "Yes",
```

```suggestion
            "yes": "Present (reason unknown or not specified)",
```

"yes" as one of values seems a bit confusing to me as dropdown value

maybe this would be better?

pinging @ZeLonewolf for English-language checking and readability

Maybe also `No` should be more expressive, like `No variable speed limit`?